### PR TITLE
01-ci: Update task 1 compile test

### PR DESCRIPTION
**Pull request by MrKevinWeiss**
Nobody actually runs the `compile_test.py` script I hope...

### CODE REVIEW

**Comment by aabadie** [comment]
Something that could also be done once the new murdock instance is in place is to modify `.murdock.yml` so that when a new tag is pushed a new job is launched. Since a new tag is pushed when a new RC is opened, then the compile/test task will be done automatically.

**Review by kaspar030 — APPROVED** [review]
ACK.

**Comment by kaspar030** [comment]
pls rebase

**Comment by kaspar030** [comment]
> Something that could also be done once the new murdock instance is in place is to modify `.murdock.yml` so that when a new tag is pushed a new job is launched. Since a new tag is pushed when a new RC is opened, then the compile/test task will be done automatically.

hm, there now is always a bors full build for the branch which can be re-used. maybe just mention that?

File: 01-ci/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Task #01 - Compile test
 =======================
 ### Description
 
-Execute ./dist/tools/compile_test/compile_test.py
+Open a dummy PR or use a backport PR on the RC and enable the `full_test` label.
```
Comment (kaspar030) [inline 01-ci/README.md:7]:
```suggestion
Check the latest release branch bors build ("staging@...").
```

**Review by kaspar030 — CHANGES_REQUESTED** [review]

File: 01-ci/README.md (hunk 1 above)
Comment (MrKevinWeiss) [reply 01-ci/README.md]: done

**Comment by MrKevinWeiss** [comment]
ping @kaspar030

File: 01-ci/README.md (hunk 1 above)
Comment (MrKevinWeiss) [reply 01-ci/README.md]: I guess this is obsolete now and should be updated to merge queues...

Comment (kaspar030) [reply 01-ci/README.md]: Yeah. Maybe make that CI agnostik: "Check the latest release branch CI build."

**Comment by Teufelchen1** [comment]
@kaspar030 can this be merged?

**Comment by MrKevinWeiss** [comment]
ping @Teufelchen1 or @mguetschow

**Review by Teufelchen1 — APPROVED** [review]

**Pull request merged**